cylc/flow/parsec/fileparse.py

```diff
@@ -239,27 +259,54 @@ def read_and_proc(fpath, template_vars=None, viewcfg=None, asedit=False):
         flines = inline(
             flines, fdir, fpath, False, viewcfg=viewcfg, for_edit=asedit)
 
+    # If empy and jinja2 sections are both filled raise an error.
+    if (all([extra_vars['empy:suite.rc'], extra_vars['jinja2:suite.rc']])):
```

Comment:
Should really handle this in cylc-rose and return something like `template_vars` so that cylc-flow doesn't need to know anything about Rose configurations (especially as they are likely to change).

Reply:
I have now moved this check into the plugin. :)